src/pyscipopt/scip.pxi

```diff
@@ -2126,6 +2140,42 @@ cdef class Model:
 
         return constraints
 
+    def getConsNVars(self, Constraint constraint):
+        cdef int nvars 
```

Comment:
Could you please add documentation for this method?

Reply:
Done! Took a little bit because I had to deal with that Git funny business we talked about yesterday, but all good now :)